addons/web_editor/static/lib/odoo-editor/src/commands/toggleList.js

```diff
@@ -19,7 +19,7 @@ HTMLElement.prototype.oToggleList = function (offset, mode = 'UL') {
         return this.parentElement.oToggleList(childNodeIndex(this));
     }
     const inLI = this.closest('li');
-    if (inLI) {
+    if (inLI && !inLI.classList.contains('nav-item')) {
```

Comment:
This makes the name of the constant weird because you expect it to be a boolean. How about `closestLi` ?

```diff
-    const inLi = this.closest('li');
-    if (inLi && !inLi.classList.contains('nav-item')) {
+    const closestLi = this.closest('li');
+    if (closestLi && !closestLi.classList.contains('nav-item')) {
```

A short comment would also be nice, to justify this ad-hoc case: we never want to mess with nav-item identation as they don't behave like regular list items.